WordPress/gutenberg
# [RNMobile] Launch support from inside editor

## Description

This feature adds the ability to create a new Customer Support ticket from the editor's "Help & Support" view. The user can also visit the general support screen from the same view (e.g. WordPress Help Center, My Tickets, etc.).

### Related PRs

- https://github.com/wordpress-mobile/gutenberg-mobile/pull/3975
- https://github.com/wordpress-mobile/WordPress-iOS/pull/17175
- https://github.com/wordpress-mobile/WordPress-Android/pull/15328

## How has this been tested?

### Test 1

1. Using the WP app and while inside the editor, tap the 3-dot menu button.
2. Tap on the "Help & Support" option.
3. In the full-screen editor help center view, tap the "Contact Support" button.
4. Observe the "Contact Support" chat window.

<img width="250" alt="Zendesk support view" src="https://user-images.githubusercontent.com/2092798/134103169-588df2b9-5491-47dd-901d-e20e3507503c.png" />

### Test 2

1. Using the WP app and while inside the editor, tap the 3-dot menu button.
2. Tap on the "Help & Support" option.
3. In the full-screen editor help center view, tap the "All Support Options" button.
4. Observe the app's support view.

<img width="250" alt="Support view for the app" src="https://user-images.githubusercontent.com/2092798/134103284-089dfe14-db6d-455c-9da3-b899f773e27a.png" />

## Screenshots
<div>
<img src="https://user-images.githubusercontent.com/2092798/135008052-9a1c8b26-6804-44c8-a536-ac4de45fefea.PNG" width="250px" alt="iOS light mode screenshot" />
<img src="https://user-images.githubusercontent.com/2092798/135008055-27cc3dcd-28c1-4c67-91be-329b88107772.PNG" width="250px" alt="iOS dark mode screenshot" />
</div>
<div>
<img src="https://user-images.githubusercontent.com/2092798/135008070-f21837ad-c623-4d15-8ea7-8864b9d98d84.png" width="250px" alt="Android light mode screenshot" />
<img src="https://user-images.githubusercontent.com/2092798/135008075-79a638d1-4e14-4a4f-b475-d63844d350b6.png" width="250px" alt="Android dark mode screenshot" />
</div>

## Types of changes
1. Adds a new component [HelpGetSupportButton](https://github.com/WordPress/gutenberg/pull/34890/files#diff-b701d34f30b835c40a2f53d0be91e3be56cc8031255846ab9b4e07f8cced5a6b) to define a styled button for the help views.
2. Triggers creating a new Support ticket when the "[Contact Support](https://github.com/WordPress/gutenberg/pull/34890/files#diff-bf50e5ac92a94cf2afdd97fcdcb1295ba8fec9cfd1bc1fdc151279999f4460f0R159)" button is tapped.
3. Triggers navigating to the main app Help options screen when the "[All Support Options](https://github.com/WordPress/gutenberg/pull/34890/files#diff-bf50e5ac92a94cf2afdd97fcdcb1295ba8fec9cfd1bc1fdc151279999f4460f0R169)" button is tapped.
4. Flows above are supported by accompanying bridge methods in the native apps.

## Checklist:
